README.md

```diff
@@ -388,6 +388,9 @@ From version 2.0 you can add 4th parameter to Message as timestamp in future. Th
 
 #### From v3 to v4
 
+- Renamed Restart to Shutdown
```

Comment:
I would add reason behind rename (either into changelog/readme or at least as commit message). Something like:

> Renamed Restart to Shutdown. Naming changed to reflect functionality of Hermes. It can gracefully stop own process, but restart (relaunch) of Hermes has to be handled by external process / library. And therefore this is shutdown and not restart.